exercises/affine-cipher/src/test/java/AffineCipherTest.java

```diff
@@ -7,130 +8,123 @@
 
 public class AffineCipherTest {
 
+    private AffineCipher affineCipher;
+
+    @Before
+    public void genAffineCipfer() {
```

Comment:
Very minor thing, should this be `genAfficeCipher`? 🙂

Reply:
yes, of course, thank you! :smile: